config/initializers/new_framework_defaults_7_2.rb

```diff
@@ -0,0 +1,70 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Looks like `load_defaults 7.2` is on, so this file isn't currently being used.

Reply:
Yeah, deleting.